sys/ios-cydia.sh

```diff
@@ -18,9 +18,13 @@ export BUILD=1
 
 if [ ! -d sys/ios-include/mach/vm_behavior.h  ]; then
 (
-	cd sys && \
-	wget http://lolcathost.org/b/ios-include.tar.gz && \
-	tar xzvf ios-include.tar.gz
+	if [ ! -f sys/ios-include.tar.gz ]; then
```

Comment:
FYI, I included `sys/ios-include.tar.gz` directly in the radare2 codebase as lolcathost was often unreachable.

Reply:
ideally we shuoldnt be distributing this, mainly because it comes from apple, so not legal and this is why its downloaded from a different source. ideally we shouldnt depend on having this tarball around

Reply:
the problem is that the host that have this binary can be reached 1 out of 10 times, so the script was not reliable. Do you have a better alternative (which does not involve me rewriting all the code related to IOS to not depend on this tar..)?